mmv1/third_party/terraform/provider/universe/universe_domain_compute_test.go

```diff
@@ -0,0 +1,112 @@
+package universe_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccUniverseDomainDisk(t *testing.T) {
+	// Skip VCR since this test can only run in specific test project.
```

Comment:
This will skip running the test in all environments, not just VCR- let's correct the comment or the skip method- if you mean to skip VCR, use `acctest.SkipIfVcr(t)`

Reply:
(same for other cases)

Reply:
Sounds good, these tests for now should be skipped in all envs.